apps/karma.conf.js

```diff
@@ -82,7 +82,7 @@ module.exports = function (config) {
       ]
     },
     mochaReporter: {
-      output: 'minimal',
+      output: envConstants.CIRCLECI ? 'minimal' : 'full',
```

Comment:
Can we also do this on the staging/test machines?  The slack output is painful too.

Reply:
Ah, good call.  What's the right set of places to display minimal vs. full output?

Reply:
+1. i think this is even more valuable for staging/test than circle